Tests/test_PDB.py

```diff
@@ -1081,6 +1083,21 @@ def test_entitity_copy(self):
             self.assertFalse(e.get_list()[0] is ee.get_list()[0])
 
 
+def eprint(*args, **kwargs):
```

Comment:
I found this function pretty useful when debugging my code since stdout is blocked when running the unit test.